src/_pytest/assertion/rewrite.py

```diff
@@ -285,7 +287,7 @@ def _rewrite_test(fn, config):
     with open(fn, "rb") as f:
         source = f.read()
     tree = ast.parse(source, filename=fn)
-    rewrite_asserts(tree, fn, config)
+    rewrite_asserts(tree, source, fn, config)
```

Comment:
Not that this is a blocker, but this will end up breaking https://pypi.org/project/pytest-ast-back-to-python.

cc @tomviner

Reply:
ah, I can restore the api and just re-read the source in the visitor using `fn`

Reply:
Just to update here, [pytest-ast-back-to-python](https://pypi.org/project/pytest-ast-back-to-python/) has now been fixed.

Reply:
Awesome Tom, thanks for the heads up!